app/services/extraction_supplying_service.rb

```diff
@@ -644,192 +689,123 @@ def get_evidence_obj(
     measure_name
   )
     evidence = {
-      'id' => '5' + '-' + record_id.to_s,
+      'id' => "5-#{record_id}",
       'status' => outcome_status,
       'identifier' => [{
         'type' => {
           'text' => 'SRDR+ Object Identifier'
         },
         'system' => 'https://srdrplus.ahrq.gov/',
-        'value' => 'Record/' + record_id.to_s
+        'value' => "Record/#{record_id}"
       }],
       'variableDefinition' => [
-        {
-          'description' => population_name,
-          'variableRole' => {
-            'coding' => [{
-              'system' => 'http://terminology.hl7.org/CodeSystem/variable-role',
-              'code' => 'population'
-            }]
+        build_variable_definition(population_name, 'population')
+      ],
+      'statistic' => [{}]
+    }
+
+    case measure_name
```

Comment:
We should probably compare case-insensitive and strip whitespace at the beginning and end. It's probably safe to do it the way you did (since these are default measures), but just to allow for more generalization and in case the user decides to add and remove and doesn't respect case or whitespace.

so do measure_name.gsub(/\s+/, " ").downcase and then change the when to lower case across.